docs/gateway/s3.md

```diff
@@ -0,0 +1,51 @@
+# Minio S3 Gateway [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+Minio S3 Gateway adds Minio features like Minio Browser and Caching to AWS S3 buckets.
+
+## Run Minio Gateway for AWS S3
+
```

Comment:
Should there be another section to explain how to connect to `AWS S3` compatible object storage servers?